src/main/java/at/hannibal2/skyhanni/test/TestCopyBestiaryValues.kt

```diff
@@ -82,14 +81,13 @@ object TestCopyBestiaryValues {
             return
         }
         val capLine = lore.nextAfter(overallProgress) ?: return
-        val rawCap = capLine.substringAfter("/").removeColor().formatNumber()
+        val rawCap = capLine.substringAfter("/").removeColor().formatLong()
```

Comment:
gets casted to an int next line
```suggestion
        val rawCap = capLine.substringAfter("/").removeColor().formatInt()
```